jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenLayerConfigurations.java

```diff
@@ -89,27 +91,39 @@ static JavaLayerConfigurations getForProject(
   }
 
   @FunctionalInterface
-  private static interface FileToLayerAdder {
+  @VisibleForTesting
+  static interface FileToLayerAdder {
 
     void add(Path sourcePath, AbsoluteUnixPath pathInContainer) throws IOException;
   }
 
-  private static void addFilesToLayer(
+  @VisibleForTesting
+  static boolean isEmptyDirectory(Path path) throws IOException {
+    if (Files.isDirectory(path)) {
+      try (Stream<Path> stream = Files.list(path)) {
+        return !stream.findAny().isPresent();
+      }
+    }
+    return false;
+  }
+
+  @VisibleForTesting
+  static void addFilesToLayer(
```

Comment:
It'd be good to have a javadoc here to explain what each of the parameters refer to.